pkgs/by-name/sy/sydbox/package.nix

```diff
@@ -25,10 +25,10 @@ rustPlatform.buildRustPackage rec {
     owner = "Sydbox";
     repo = "sydbox";
     rev = "refs/tags/v${version}";
```

Comment:
```suggestion
    tag = "v${version}";
```

Reply:
I completely missed the `tag` argument being exposed last year.